### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by solomkao**
pull request from develop to master
Main class
Author class
Book class
User class
and dome config files

### CODE REVIEW

File: src/main/resources/script.sql [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
+         join authors_books ab on books.id = ab.bookid
+         join authors a on a.id = ab.authorid
+         join users_books ub on books.id = ub.bookid
+where ub.userid = 1;
+
+select books.title, authors.name, authors.surname from books, authors, users, authors_books, users_books where
+        userid = 1 and users_books.userid = userid and users_books.bookid
+        = books.id and authors_books.authorid = authors.id and authors_books.bookid = books.id
+
+//garbage
```
Comment (illia-nerdysoft) [inline src/main/resources/script.sql]: if it is garbage, it should be removed)

File: src/main/resources/script.sql [hunk 2]
```diff
@@ -1,32 +1,57 @@
...
 
-create table library_schema.authors_books(
-    id serial primary key,
-    authorId integer,
-    bookId integer,
-    foreign key (authorId) references library_schema.authors(id),
-    foreign key (bookId) references library_schema.books(id)
-)
\ No newline at end of file
+create table users_books(
```
Comment (illia-nerdysoft) [inline src/main/resources/script.sql]: you have 2 tables with the same name. check it please)

File: src/main/java/com/solomka/User.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
```
Comment (illia-nerdysoft) [inline src/main/java/com/solomka/User.java]: you can avoid using column annotation if you don't need to change the column name) so, you can remove it out of here. apply this rule everywhere

**Review by illia-nerdysoft — CHANGES_REQUESTED** [review]

File: src/main/resources/script.sql (hunk 1 above)
Comment (solomkao) [reply src/main/resources/script.sql]: done

File: src/main/resources/script.sql (hunk 2 above)
Comment (solomkao) [reply src/main/resources/script.sql]: done

File: src/main/java/com/solomka/User.java (hunk 3 above)
Comment (solomkao) [reply src/main/java/com/solomka/User.java]: done

**Review by illia-nerdysoft — APPROVED** [review]

**Pull request merged**